### PR TITLE
(DOCS-28243): Add Kotlin link to Custom Function Auth page

### PR DESCRIPTION
**Pull request by cbullinger**
## Pull Request Info

- Add link to Authenticate Users - Kotlin SDK page for new Custom Function authentication
- Replace tabbed, hardcoded examples with links to respective SDK docs

### Jira

- https://jira.mongodb.org/browse/DOCSP-28243

### Staged Changes

- [Custom Function Authentication](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/docsp-28243-kotlin-custom-functions/authentication/custom-function/)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-realm update(s), if any: https://jira.mongodb.org/browse/DOCSP-26563 
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/authentication/custom-function.txt [hunk 1]
```diff
@@ -79,75 +79,18 @@ included with the custom function provider credential in the client app.
 The function accepts any value provided from your client app, so the
 actual field names and values depend on your implementation.
 
-.. example::
+For examples of how to create a custom credential ``payload`` object
+using the Realm SDKs, refer to the documentation for a specific SDK:
```
Comment (mongodben) [inline source/authentication/custom-function.txt:83]:
```suggestion
using the Realm SDKs, refer to the documentation for each SDK:
```

File: source/authentication/custom-function.txt [hunk 2]
```diff
@@ -79,75 +79,18 @@ included with the custom function provider credential in the client app.
...
-            Map<String, String> credentials = {
-               "username": "someUsername",
-               "pin": "24601"
-            };
-            // payload must be a JSON-encoded string
-            String payload = jsonEncode(credentials);
-
-            Credentials customCredentials = Credentials.function(payload);
-            User currentUser = await app.logIn(customCredentials);
+- :ref:`C++ SDK (Alpha) <cpp-login-custom-function>`
```
Comment (mongodben) [inline source/authentication/custom-function.txt:85]:
```suggestion
- :ref:`C++ SDK <cpp-login-custom-function>`
```

nit more maintainable to not include "Alpha" here for when it's updated to beta, rc, ga, etc

**Review by mongodben — APPROVED** [review]
lgtm! 2 small things

File: source/authentication/custom-function.txt (hunk 2 above)
Comment (cbullinger) [reply source/authentication/custom-function.txt:85]: Good catch, thanks!

**Pull request merged**